java-client/session/src/main/java/io/deephaven/client/impl/Session.java

```diff
@@ -32,8 +32,8 @@ public interface Session
     // ----------------------------------------------------------
 
     /**
-     * Advanced usage, creates a new export ID for {@code this} session, but must be managed by the caller. Useful for
-     * more advanced integrations, particularly around doPut. Callers are responsible for {@link #release(ExportId)
+     * Advanced usage, creates a new table export ID for {@code this} session, but must be managed by the caller.Useful
```

Comment:
probably should put that removed space back - but why can't this be used for non-table exports any more?

Reply:
The existing implementation is really catered around "Table"; mostly a lack of other use cases or naivety when these interfaces were created. To correct this in the future if necessary, we can add:

```java
ExportId newExportId(String type);
```